manifests/Iron.Star.yaml

```diff
@@ -0,0 +1,11 @@
+name: Iron Star Corp
+authors: Alaric Atreus
+homepage: https://github.com/ThunderOS/Iron_Star_Corp_Endless_Sky
+license: CC-BY-SA-4.0
+version: v0.1.01
```

Comment:
```suggestion
version: Iron.Star
```
The version is the tag, so if you say v0.1.01, then $version will give the wrong thing.